### ISSUE TITLE
Misuse of xacro name

### ISSUE THREAD

**Issue by manuelgitgomes** [lardemua/atom#774]
Hello @miguelriemoliveira.

I think we have misused the "xacro" name a lot.
After calibration we generate an URDF that does not use any xacro features, such as macros or properties.

I think it would be less confusing to call it URDF.

**Comment by miguelriemoliveira** [comment]
Right. I agree, we often are reffering to the URDF.

**Comment by miguelriemoliveira** [comment]
I agree, but what should we do? Is there anything to do on this issue or should we close?

**Comment by manuelgitgomes** [comment]
> I agree, but what should we do? Is there anything to do on this issue or should we close?

We should change the generated URDF to be only .urdf and not .urdf.xacro (the inital_estimate and the optimized one).
Some prints also state xacro instead of urdf.

**Comment by miguelriemoliveira** [comment]
Right. Makes sense. Can you do it?
